Doc/library/dis.rst

```diff
@@ -699,8 +699,8 @@ iterations of the loop.
 
    Yields ``STACK.pop()`` from a :term:`generator`.
 
-    .. versionchanged:: 3.11
-       oparg set to be the stack depth, for efficient handling on frames.
+    .. versionchanged:: 3.12
+       oparg set to be the exception block depth, for efficient closing of generators.
```

Comment:
Shouldn't the 3.11 note be retained (assuming it's truthful)?

Reply:
done